scipy/optimize/_highs/cython/src/HighsSparseMatrix.pxd

```diff
@@ -0,0 +1,16 @@
+# distutils: language=c++
+# cython: language_level=3
+
+from libcpp.vector cimport vector
+
+from .HConst cimport MatrixFormat
+
+
+cdef extern from "HighsSparseMatrix.h" nogil:
```

Comment:
The raw CSC constraint matrix variables were abstracted into this new `HighsSparseMatrix` class